### DISCUSSION TITLE
Feature Request: Allow fully disabling certain components

### DISCUSSION THREAD

**Discussion by nahtnam** [saadeghi/daisyui#535] [Ideas / feature requests]
Hello,

I love daisy and fully intend to use it in my projects. However, I'm also a paying customer of Tailwind UI. In some instances, I would like to replace `btn` with a tailwind UI alternative. I noticed that using `@apply` like this:

```
.btn {
  @apply rounded-full;
}
```

is additive. This means that `rounded-full` gets added onto the existing button. I was hoping there would be a config option such as `disabledComponents: ["btn"]` where no button styles classes get included. This would allow me to write my own button classes, but continue to use other things such as card

**Comment by saadeghi** [comment]
This would be nice but currently there are no plans for adding this so I can't promise anything.  

I would appreciate if anyone wants to contribute for adding this feature

**Comment by saadeghi** [comment]
I'm moving this to discussions

**Comment by jtormey** [comment]
This is solved by https://daisyui.com/docs/config/#exclude

**Reply by saadeghi** [reply]
Yes. This discussion is from 3 years ago.

**Reply by jtormey** [reply]
Just leaving a comment for anyone else that is looking for this. I came across this discussion before finding the solution in the docs (searching for "include" or "exclude" returns no results).
